source/core/coverage/ut_coverage.pkb

```diff
@@ -94,6 +96,15 @@ create or replace package body ut_coverage is
               on s.name  = f.object_name
              and s.type  = f.object_type
              and s.owner = f.object_owner';
+    elsif a_coverage_options.include_schema_expr is not null or a_coverage_options.include_object_expr is not null then
+      l_full_name := q'[lower(s.type||' '||s.owner||'.'||s.name)]';
+      if a_coverage_options.include_schema_expr is not null then
+        l_filters := q'[and regexp_like(s.owner,']'||a_coverage_options.include_schema_expr||q'[','i')]';
+      end if;
+      
+      if a_coverage_options.include_object_expr is not null then
+        l_filters := l_filters|| q'[ and regexp_like(s.name,']'||a_coverage_options.include_object_expr||q'[','i')]';    
```

Comment:
This expression cannot be concatenated as it opens up a security hole for SQL injection.
We need to use a bind variable to pass that value to the filtered query.